docs/antctl.md

```diff
@@ -181,12 +183,25 @@ antrea-network-policy.md#antrea-native-policy-ordering-based-on-priorities).
 antctl get networkpolicy --sort-by=effectivePriority
 ```
 
-Antrea Agent additionally supports printing NetworkPolicies applied to a
-specified local Pod using this `antctl` command:
+Antrea Agent additionally supports some functions using this `antctl` command.
```

Comment:
"Antrea Agent supports some extra `antctl` commands"?

Reply:
Custom handlers are only in Agent.